Error when importing the sassdoc generated files from the 7.3.1 branch.

## Description  
An error is shown when I try to build the sassdoc documentation with the previously exported json files

 * igniteui-angular version: 7.3.1
 * browser: 

## Steps to reproduce  

1. Export the documentation from the 7.3.1 branch of the igniteui-angular repository with "npm run sassdoc-build:export".
2. Import the files to generate the documentation with "npm run sassdoc-build:import"
3. ...

## Result  
There's an error:
» [ERROR] TypeError: Cannot read property 'trim' of undefined

## Expected result  
There shouldn't be an error and the documentation should be build normally.

--- a/extras/docs/themes/sassdoc/index.js
+++ b/extras/docs/themes/sassdoc/index.js
@@ -124,6 +124,10 @@ const theme = themeleon(__dirname, function (t) {
                 return config ? config.versions : '';
             },
             getLang: () => {
+                if (!process.env.SASSDOC_LANG) {
+                    return;
+                }
+
                 return process.env.SASSDOC_LANG.trim();
             },
             ifCond: (v1, operator, v2, options) => {

--- a/extras/docs/themes/sassdoc/index.js
+++ b/extras/docs/themes/sassdoc/index.js
@@ -124,7 +124,7 @@ const theme = themeleon(__dirname, function (t) {
                 return config ? config.versions : '';
             },
             getLang: () => {
-                return process.env.SASSDOC_LANG;
+                return process.env.SASSDOC_LANG.trim();
             },
             ifCond: (v1, operator, v2, options) => {
                 switch (operator) {

--- a/extras/docs/themes/sassdoc/views/index.hbs
+++ b/extras/docs/themes/sassdoc/views/index.hbs
@@ -8,13 +8,14 @@
 
         {{#ifCond lang '==' 'en'}}
             <link rel="canonical" href="{{baseURl 'sassdoc_default_url' 'en'}}" />
-            <link rel="alternate" href="{{baseURl 'sassdoc_default_url' 'en'}}" hreflang="en" />
-            <link rel="alternate" href="{{baseURl 'sassdoc_default_url' 'en'}}" hreflang="en-us" />
         {{else}}
             <link rel="canonical" href="{{baseURl 'sassdoc_default_url' 'jp'}}" />
-            <link rel="alternate" href="{{baseURl 'sassdoc_default_url' 'jp'}}" hreflang="ja" />
-            <link rel="alternate" href="{{baseURl 'sassdoc_default_url' 'jp'}}" hreflang="ja-jp" />
         {{/ifCond}}
+        <link rel="alternate" href="{{baseURl 'sassdoc_default_url' 'en'}}" hreflang="en" />
+        <link rel="alternate" href="{{baseURl 'sassdoc_default_url' 'en'}}" hreflang="en-us" />
+        <link rel="alternate" href="{{baseURl 'sassdoc_default_url' 'jp'}}" hreflang="ja" />
+        <link rel="alternate" href="{{baseURl 'sassdoc_default_url' 'jp'}}" hreflang="ja-jp" />
+        <link rel="alternate" href="{{baseURl 'sassdoc_default_url' 'en'}}" hreflang="x-default" />
 
 
         <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">

--- a/extras/docs/themes/sassdoc/views/index.hbs
+++ b/extras/docs/themes/sassdoc/views/index.hbs
@@ -1,18 +1,21 @@
 <!DOCTYPE html>
-<html lang="{{getLang}}">
+<html lang="{{lang}}">
     <head>
         <meta charset="utf-8" />
         <meta http-equiv="X-UA-Compatible" content="IE=edge">
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <title>Ignite UI for Angular | Sass Documentation</title>
 
-        <link rel="canonical" href="{{baseURl 'sassdoc_default_url' 'en'}}" />
-        <link rel="canonical" href="{{baseURl 'sassdoc_default_url' 'jp'}}" />
+        {{#ifCond lang '==' 'en'}}
+            <link rel="canonical" href="{{baseURl 'sassdoc_default_url' 'en'}}" />
+            <link rel="alternate" href="{{baseURl 'sassdoc_default_url' 'en'}}" hreflang="en" />
+            <link rel="alternate" href="{{baseURl 'sassdoc_default_url' 'en'}}" hreflang="en-us" />
+        {{else}}
+            <link rel="canonical" href="{{baseURl 'sassdoc_default_url' 'jp'}}" />
+            <link rel="alternate" href="{{baseURl 'sassdoc_default_url' 'jp'}}" hreflang="ja" />
+            <link rel="alternate" href="{{baseURl 'sassdoc_default_url' 'jp'}}" hreflang="ja-jp" />
+        {{/ifCond}}
 
-        <link rel="alternate" href="{{baseURl 'sassdoc_default_url' 'en'}}" hreflang="en" />
-        <link rel="alternate" href="{{baseURl 'sassdoc_default_url' 'en'}}" hreflang="en-us" />
-        <link rel="alternate" href="{{baseURl 'sassdoc_default_url' 'jp'}}" hreflang="ja" />
-        <link rel="alternate" href="{{baseURl 'sassdoc_default_url' 'jp'}}" hreflang="ja-jp" />
 
         <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
         <link rel="stylesheet" href="https://infragistics.com/assets/modern/css/layout.css">
@@ -54,6 +57,7 @@
     <!-- End Google Tag Manager (noscript) -->
 
         {{> header}}
+        <div>{{getLang}}</div>
 
         <script src="assets/js/versioning/tag-versions.req.js"></script>
     {{!

--- a/extras/docs/themes/typedoc/src/layouts/default.hbs
+++ b/extras/docs/themes/typedoc/src/layouts/default.hbs
@@ -9,13 +9,14 @@
 
     {{#ifCond settings.localize '===' 'en'}}
         <link rel="canonical" href="{{getConfigData 'typedoc_default_url' 'en'}}{{url}}" />
-        <link rel="alternate" href="{{getConfigData 'typedoc_default_url' 'en'}}{{url}}" hreflang="en" />
-        <link rel="alternate" href="{{getConfigData 'typedoc_default_url' 'en'}}{{url}}" hreflang="en-us" />
     {{else}}
         <link rel="canonical" href="{{getConfigData 'typedoc_default_url' 'jp'}}{{url}}" />
-        <link rel="alternate" href="{{getConfigData 'typedoc_default_url' 'jp'}}{{url}}" hreflang="ja-jp" />
-        <link rel="alternate" href="{{getConfigData 'typedoc_default_url' 'jp'}}{{url}}" hreflang="ja" />
     {{/ifCond}}
+    <link rel="alternate" href="{{getConfigData 'typedoc_default_url' 'en'}}{{url}}" hreflang="en" />
+    <link rel="alternate" href="{{getConfigData 'typedoc_default_url' 'en'}}{{url}}" hreflang="en-us" />
+    <link rel="alternate" href="{{getConfigData 'typedoc_default_url' 'jp'}}{{url}}" hreflang="ja" />
+    <link rel="alternate" href="{{getConfigData 'typedoc_default_url' 'jp'}}{{url}}" hreflang="ja-jp" />
+    <link rel="alternate" href="{{getConfigData 'typedoc_default_url' 'en'}}{{url}}" hreflang="x-default" />
 
     <link rel="stylesheet" href="https://infragistics.com/assets/modern/css/layout.css">
     <link rel="stylesheet" href="https://infragistics.com/assets/modern/css/animate-custom.css">

--- a/extras/docs/themes/typedoc/src/layouts/default.hbs
+++ b/extras/docs/themes/typedoc/src/layouts/default.hbs
@@ -7,13 +7,15 @@
     <meta name="description" content="">
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
-    <link rel="canonical" href="{{getConfigData 'typedoc_default_url' 'en'}}{{url}}" />
-    <link rel="canonical" href="{{getConfigData 'typedoc_default_url' 'jp'}}{{url}}" />
-
-    <link rel="alternate" href="{{getConfigData 'typedoc_default_url' 'en'}}{{url}}" hreflang="en" />
-    <link rel="alternate" href="{{getConfigData 'typedoc_default_url' 'en'}}{{url}}" hreflang="en-us" />
-    <link rel="alternate" href="{{getConfigData 'typedoc_default_url' 'jp'}}{{url}}" hreflang="ja" />
-    <link rel="alternate" href="{{getConfigData 'typedoc_default_url' 'jp'}}{{url}}" hreflang="ja-jp" />
+    {{#ifCond settings.localize '===' 'en'}}
+        <link rel="canonical" href="{{getConfigData 'typedoc_default_url' 'en'}}{{url}}" />
+        <link rel="alternate" href="{{getConfigData 'typedoc_default_url' 'en'}}{{url}}" hreflang="en" />
+        <link rel="alternate" href="{{getConfigData 'typedoc_default_url' 'en'}}{{url}}" hreflang="en-us" />
+    {{else}}
+        <link rel="canonical" href="{{getConfigData 'typedoc_default_url' 'jp'}}{{url}}" />
+        <link rel="alternate" href="{{getConfigData 'typedoc_default_url' 'jp'}}{{url}}" hreflang="ja-jp" />
+        <link rel="alternate" href="{{getConfigData 'typedoc_default_url' 'jp'}}{{url}}" hreflang="ja" />
+    {{/ifCond}}
 
     <link rel="stylesheet" href="https://infragistics.com/assets/modern/css/layout.css">
     <link rel="stylesheet" href="https://infragistics.com/assets/modern/css/animate-custom.css">

